Sources_v3/Database/DatabaseContainer.swift

```diff
@@ -16,13 +16,24 @@ class DatabaseContainer: NSPersistentContainer {
         case onDisk(databaseFileURL: URL)
     }
     
+    /// We use `writableContext` for having just one place to save changes
+    /// so it’s not possible to have conflicts when saving payloads from various sources.
+    /// All writes are happening serially using this context and its `write { }` methods.
```

Comment:
... off topic: when writing this, I felt "seriously" wasn't the right word. "Serially" was the word :D

Reply:
I imagined some angry middle aged worker, named "Context", writing entities into a big excel file with a very serious attitude on his face